framework/src/modules/chain/bft/finality_manager.js

```diff
@@ -88,7 +88,7 @@ class FinalityManager extends EventEmitter {
 
 		debug('after adding block header', {
 			finalizedHeight: this.finalizedHeight,
-			prevotedConfirmedHeight: this.prevotedConfirmedHeight,
+			chainMaxHeightFinalized: this.chainMaxHeightFinalized,
```

Comment:
Its duplicate information both `finalizedHeight` and `chainMaxHeightFinalized` are the same attribute.